quickstarts/microsoft.authorization/rbac-managedidentity/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "managedIdentityName": {
+      "value": "RbacManagedIdentityName"
+    },
+    "location": {
```

Comment:
remove this and add a defaultValue of resourceGroup().location to the location param